### PR TITLE
Correct the isotropic Poisson's ratio range

### PR DESCRIPTION
**Pull request by sytuannguyen**
This PR corrects the range of the isotropic Poisson's ratio in LinearElasticIsotropic constitutive model. 

The isotropic Poisson's ratio is actually limited to a wrong range: nu >= 0. 
It is now corrected to the valid range: -0.5 < nu < 0.5

The range of the elastic bulk, shear and Young moduli are also corrected from >=0 to >0 to avoid the division to zero. For example, the calculation of the Poisson's ratio will not accept the value K = 0:

nu = 0.5 * (1 - E / 3 / K ) 

You can find also some minor cleanup

### CODE REVIEW

File: src/coreComponents/constitutive/solid/ElasticIsotropic.cpp [hunk 1]
```diff
@@ -44,27 +44,25 @@ ElasticIsotropic::ElasticIsotropic( string const & name, Group * const parent ):
   registerWrapper< real64 >( viewKeyStruct::defaultYoungsModulusString )->
     setApplyDefaultValue( -1 )->
     setInputFlag( InputFlags::OPTIONAL )->
-    setDescription( "Elastic Young's Modulus." );
+    setDescription( "Elastic Young's Modulus Parameter" );
```
Comment (joshua-white) [inline src/coreComponents/constitutive/solid/ElasticIsotropic.cpp:47]:
```suggestion
    setDescription( "Default Young's Modulus" );
```

File: src/coreComponents/constitutive/solid/ElasticIsotropic.cpp [hunk 2]
```diff
@@ -44,27 +44,25 @@ ElasticIsotropic::ElasticIsotropic( string const & name, Group * const parent ):
...
 
   registerWrapper< real64 >( viewKeyStruct::defaultPoissonRatioString )->
     setApplyDefaultValue( -1 )->
     setInputFlag( InputFlags::OPTIONAL )->
-    setDescription( "Poisson's ratio" );
+    setDescription( "Elastic Poisson's ratio Parameter" );
```
Comment (joshua-white) [inline src/coreComponents/constitutive/solid/ElasticIsotropic.cpp:52]:
```suggestion
    setDescription( "Default Poisson's Ratio" );
```

**Review by joshua-white — APPROVED** [review]
I added minor suggestions, but this one looks good to merge when you are ready.

**Comment by sytuannguyen** [comment]
@joshua-white I leave the change YoungsMod to YoungMod for other PR because it touches a list of file including documents that is out of the scope of this tiny PR

**Comment by sytuannguyen** [comment]
@joshua-white now It is ready to be merged

**Comment by sytuannguyen** [comment]
@joshua-white This will be merged this tiny PR to #1323 to faciliate the update with develop

**Pull request closed without merging**